timemachine/fe/rbfe.py

```diff
@@ -94,6 +95,7 @@ def setup_in_env(
     """Prepare potentials, concatenate environment and ligand coords, apply HMR, and construct barostat"""
     barostat_interval = 25
     system = st.combine_with_host(host.system, lamb, host.num_water_atoms, st.ff, host.omm_topology)
+    assert_bonds_defined_for_chiral_volumes(system)
```

Comment:
Nice.

Perhaps in a later stage we could expand this into a function `validate_hybrid_params(system)` or something that includes this assertion (`HarmonicBond(i, j).force_constant > DEFAULT_BOND_IS_PRESENT_K` whenever a chiral volume includes center atom `i` and non-center atom `j`) in addition to other assertions, such as the ones you suggested in https://github.com/proteneer/timemachine/pull/1406/files#r1831720012 .